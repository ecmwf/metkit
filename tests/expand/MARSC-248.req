retrieve,
    levtype=pl,
    level=1000/500,
    date=20250416,
    param=t,
    accuracy=LOW,
    target=data.reference 