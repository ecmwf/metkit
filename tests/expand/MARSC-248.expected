RETRIEVE,
    CLASS      = OD,
    TYPE       = AN,
    STREAM     = OPER,
    EXPVER     = 0001,
    REPRES     = SH,
    LEVTYPE    = PL,
    LEVELIST   = 1000/500,
    PARAM      = 130,
    DATE       = 20250416,
    TIME       = 1200,
    STEP       = 0,
    DOMAIN     = G,
    TARGET     = data.reference,
    ACCURACY   = 8 